dotcom-rendering/src/web/browser/islands/doHydration.ts

```diff
@@ -44,22 +48,16 @@ export const doHydration = async (
 			element.setAttribute('data-gu-ready', 'true');
 			const timeTaken = end();
 
-			return { clientOnly, timeTaken };
+			return { clientOnly, timeTaken, timeToLoad };
 		})
-		.then(({ clientOnly, timeTaken }) => {
+		.then(({ clientOnly, timeTaken, timeToLoad }) => {
 			if (!('getEntriesByType' in window.performance)) return;
 
-			// Log performance info
-			const { duration: download = -1 } =
-				window.performance
-					.getEntriesByType('resource')
-					.find((p) => p.name.includes(`/${name}-importable.`)) ?? {};
```

Comment:
I wonder if it's worth keeping this as the browser's measurement of the download time of the module? Maybe as an additional logged out metric?

Reply:
As an aside, I didn't know you could use the perfomance API to do this, very cool!

Reply:
This API does not work very reliably in all browsers and can fail to report meaningful data when the module is cached.

https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming